src/rand_tangent.jl

```diff
@@ -11,11 +13,18 @@ rand_tangent(rng::AbstractRNG, x::AbstractString) = NoTangent()
 
 rand_tangent(rng::AbstractRNG, x::Integer) = NoTangent()
 
-rand_tangent(rng::AbstractRNG, x::T) where {T<:Number} = randn(rng, T)
+# Try and make nice numbers with short decimal representations for good error messages
+# while also not biasing the sample space too much
+function rand_tangent(rng::AbstractRNG, x::T) where {T<:Number}
+    return round(8randn(rng, T), sigdigits=5, base=2)
```

Comment:
```suggestion
    return round(randn(rng, T), sigdigits=5, base=2)
```
Or is there a reason why we multiply by 8? Maybe adding a comment would be helpful?

Reply:
```suggestion
    # multiply by 9 to give a bigger range of values tested: no so tightly clustered around 0.
    return round(9randn(rng, T), sigdigits=5, base=2)
```

Reply:
please can you write it as `9 * randn` so it's easier to see?